src/commons/controlBar/ControlBarShareButton.tsx

```diff
@@ -142,4 +182,30 @@ export class ControlBarShareButton extends React.PureComponent<ControlBarShareBu
       this.shareInputElem.current.select();
     }
   }
+
+  private fetchUUID(tokens: Tokens) {
```

Comment:
We should adhere to the existing exposed functions for making requests (e.g. in `RequestsSaga.ts`)